htdocs/web_portal/views/site/view_site.php

```diff
@@ -649,6 +650,32 @@ class="header"
                             }
                             ?>
                         </td>
+                        <td style="text-align:center">
+                            <?php
+                            $useTime = $APIAuthEnt->getLastRenewTime();
+                            $titleStr = 'Last renewed ' . $useTime->format('d-m-Y H:iTP');
+                            echo '<div title="' . $titleStr . '">';
+                            ?>
+                            <?php if (!$portalIsReadOnly) :?>
+                            <form action="index.php?Page_Type=Edit_API_Authentication_Entity&amp;
+                                          authentityid=<?php echo $APIAuthEnt->getId();?>&amp;
+                                          isRenewalRequest=true"
+                                   method="post">
+                                <button type="submit">
+                                <?php
+                                    echo '<img height="18px" src="'
+                                        . \GocContextPath::getPath()
+                                        . 'img/refresh_key.png"';
+                                    echo 'title="Renew the API credentials"' . '/>';
```

Comment:
```suggestion
                                    echo 'title="Renew the API credential"' . '/>';
```

Reply:
[Removed from codebase] This is no longer relevant because of the design change.